### PR TITLE
[4.0] [plg_system_skipto] Simplify code

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Simplifies code by storing meaningful parameter values.

### Testing Instructions

Apply patch.
Try different 'Site Section' options in plugin.
Test that selected section option is respected (plugin runs only on site, administrator or both based on option).

### Expected result

Works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: plugins/system/skipto/skipto.php [hunk 1]
```diff
@@ -37,9 +38,9 @@ class PlgSystemSkipto extends CMSPlugin
 	 */
 	public function onAfterDispatch()
 	{
-		$section = $this->params->get('section_skipto', 2);
+		$section = $this->params->get('section', 'administrator');
 
-		if ($section !== 3 && ($this->app->isClient('administrator') && $section !== 2 || $this->app->isClient('site') && $section !== 1))
+		if ($section !== 'both' && $this->app->isClient($section) !== true)
 		{
```
Comment (C-Lodder) [inline plugins/system/skipto/skipto.php:44]: `!$this->app->isClient($section)`?

Comment (SharkyKZ) [reply plugins/system/skipto/skipto.php:44]: When possible, we should use type-safe comparisons.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3effe5a35cc4d20bd733db1ff87f465d57db1fd1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25687">issues.joomla.org/tracker/joomla-cms/25687</a>.</sub>

**Comment by richard67** [comment]
I really would like to test that, but I don't see that skip to menu, neither in admin nor on site. Plugin is enabled. Am I doing something wrong?

**Comment by SharkyKZ** [comment]
I think you have to press Tab. Or maybe 0. If none of the keys work, just inspect the page's source code to check that assets are loaded on correct sections of the site. About the rest complain to Brain.

**Comment by richard67** [comment]
Ahh now I know what brian meant with "tabbing" in his description when he added that skipto thing. In a browser which has tabs I have expected something else. Will finish eating and then test your PR. Thanks for explaining.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 3effe5a35cc4d20bd733db1ff87f465d57db1fd1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25687">issues.joomla.org/tracker/joomla-cms/25687</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25687">issues.joomla.org/tracker/joomla-cms/25687</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**